website/docs/r/registration_assignment.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+subcategory: "Managed Services"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_registration_assignment"
+description: |-
+    Assigns a given Registration Definition to a subscription or a resource group.
+
+---
+
+# azurerm_registration_definition
```

Comment:
I will change this to azurerm_registration_assignment in the next round of changes.